src/CollapsibleBlock/CollapsibleBlock.tsx

```diff
@@ -0,0 +1,26 @@
+import { NodeViewContent, NodeViewWrapper } from '@tiptap/react';
+import './styles.css';
+
+export const CollapsibleBlock = (props: any) => {
+    const handleButtonClick = () => {
+        props.updateAttributes({
+            folded: !props.node.attrs.folded,
+        });
+    };
+    return (
+        <NodeViewWrapper>
+            <div className='draggable-item collapsible-block' style={{ display: 'flex', flexDirection: 'row', alignItems: 'flex-start' }}>
+                <div
+                    className="drag-handle"
+                    contentEditable="false"
+                    draggable="true"
+                    data-drag-handle
+                    style={{ backgroundImage: `url('data:image/svg+xml;charset=UTF-8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 10 16"><path fill-opacity="0.2" d="M4 14c0 1.1-.9 2-2 2s-2-.9-2-2 .9-2 2-2 2 .9 2 2zM2 6C.9 6 0 6.9 0 8s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0-6C.9 0 0 .9 0 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm6 4c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z" /></svg>')` }}
```

Comment:
Factor this SVG string out into a variable that explains what it is, e.g.

```ts
const DRAG_HANDLE_SVG = `url(...)`;
```

or put it in an SVG file and `import` that (it should be possible to get this to work with both vite and TypeScript).